Kickstarter-iOS/DataSources/BackerDashboardPagesDataSource.swift

```diff
@@ -7,12 +7,10 @@ internal final class BackerDashboardPagesDataSource: NSObject, UIPageViewControl
 
   internal init(delegate: UIViewController, sort: DiscoveryParams.Sort) {
     let backedController = BackerDashboardProjectsViewController()
-    let savedController = BackerDashboardProjectsViewController()
+    backedController.configureWith(projectsType: .backed, sort: sort)
```

Comment:
we typically use that style when we are pulling something from a storyboard. This controller has no storyboard representation.